[Bug] Errors on installing go-prompt via go get in Ubuntu 18

# Bug reports

Got the errors in the system terminal on installing the **go-prompt** package via command
`go get github.com/c-bata/go-prompt` in Ubuntu 18.

## Expected Behavior

Package has been downloaded and installed, can be used for program build (i.e. via go run, go build).

## Current Behavior and Steps to Reproduce

On installing via the command `go get github.com/c-bata/go-prompt` in Ubuntu 18 (and in docker with images golang:1.13 or golang:1.14)  got the errors

`
../../c-bata/go-prompt/internal/term/raw.go:27:75: cannot use (*syscall.Termios)(&n) (type *syscall.Termios) as type *unix.Termios in argument to termios.Tcsetattr
`
`
../../c-bata/go-prompt/internal/term/term.go:22:40: cannot use &saveTermios (type *syscall.Termios) as type *unix.Termios in argument to termios.Tcgetattr
`
`
../../c-bata/go-prompt/internal/term/term.go:33:68: cannot use &o (type *syscall.Termios) as type *unix.Termios in argument to termios.Tcsetattr
`

Tried to use go modules, and install a particular release (tag), alike 
`go get -v github.com/c-bata/go-prompt@v0.2.0` but none of them helps (tried all 6 releases) -  no changes, the same errors.

At the same time, in the Windows WSL with Ubuntu 18 the package is being installed w/o errors and can be used as expected.

## Context

* Operating System: Ubuntu 18 (Linux 5.0.0-38-generic #41-Ubuntu SMP Tue Dec 3 00:27:35 UTC 2019 x86_64 x86_64 x86_64 GNU/Linux); docker images golang:1.13 or golang:1.14
* Go version: on Ubuntu 18 - where errors occur: 1.14.1; on Windows WSL / Ubuntu 18: 1.14
* Terminal Emulator: none (fails on installing step)
* tag: any release

Thanks for your help!

--- a/go.mod
+++ b/go.mod
@@ -5,6 +5,6 @@ require (
 	github.com/mattn/go-isatty v0.0.3 // indirect
 	github.com/mattn/go-runewidth v0.0.3
 	github.com/mattn/go-tty v0.0.0-20180219170247-931426f7535a
-	github.com/pkg/term v0.0.0-20180423043932-cda20d4ac917
-	golang.org/x/sys v0.0.0-20180620133508-ad87a3a340fa
+	github.com/pkg/term v1.0.0
+	golang.org/x/sys v0.0.0-20200519105757-fe76b779f299
 )

--- a/internal/term/raw.go
+++ b/internal/term/raw.go
@@ -6,6 +6,7 @@ import (
 	"syscall"
 
 	"github.com/pkg/term/termios"
+	"golang.org/x/sys/unix"
 )
 
 // SetRaw put terminal into a raw mode
@@ -24,5 +25,5 @@ func SetRaw(fd int) error {
 	n.Cc[syscall.VMIN] = 1
 	n.Cc[syscall.VTIME] = 0
 
-	return termios.Tcsetattr(uintptr(fd), termios.TCSANOW, (*syscall.Termios)(&n))
+	return termios.Tcsetattr(uintptr(fd), termios.TCSANOW, (*unix.Termios)(&n))
 }

--- a/internal/term/term.go
+++ b/internal/term/term.go
@@ -4,18 +4,18 @@ package term
 
 import (
 	"sync"
-	"syscall"
 
 	"github.com/pkg/term/termios"
+	"golang.org/x/sys/unix"
 )
 
 var (
-	saveTermios     syscall.Termios
+	saveTermios     unix.Termios
 	saveTermiosFD   int
 	saveTermiosOnce sync.Once
 )
 
-func getOriginalTermios(fd int) (syscall.Termios, error) {
+func getOriginalTermios(fd int) (unix.Termios, error) {
 	var err error
 	saveTermiosOnce.Do(func() {
 		saveTermiosFD = fd

